app/views/layouts/_cookie_banner.html.erb

```diff
@@ -0,0 +1,6 @@
+<div id="cookieBanner" class="hidden">
+    By continuing to visit this site, you accept the use of cookies by Google Analytics for statistical purposes.
```

Comment:
Can we make it so we don't collect stats if the user continues to use the site without selecting "Accept"?

I'm not a fan of sites that take ignoring the button as implicit consent.

If this is already the case, can we rephrase the part that implies data is being collected just by "continuing to visit this site"?

Reply:
Whoops. That was a bad copy/paste decision. I agree, that's a Very Bad approach, and not one that's reflected in the code.